Rentit-backend/config.env

```diff
@@ -3,4 +3,5 @@ PORT=2000
 DATABASE =mongodb+srv://a5h101:<PASSWORD>@cluster0.cyfx5wk.mongodb.net/Rentit?retryWrites=true&w=majority
```

Comment:
Add `config.env` to `.gitignore` and delete from github. It contains sensitive details you should not make it available on GitHub.